src/shipping/components/ShippingZoneDetailsPage/ShippingZoneDetailsPage.tsx

```diff
@@ -33,9 +35,27 @@ import ShippingZoneWarehouses from "../ShippingZoneWarehouses";
 
 export interface FormData extends MetadataFormData {
   name: string;
+  description: string;
   warehouses: string[];
 }
 
+const maxDescriptionLength = 300;
```

Comment:
Nothing big, but maybe when making consts outside of the component function it'd be easier to tell apart from local component consts if the const was like so MAX_DESCRIPTION_LENGTH?